ts/WoltLabSuite/Core/Ui/Page/Menu/Provider.ts

```diff
@@ -0,0 +1,13 @@
+export interface PageMenuProvider {
```

Comment:
Missing module level comment. This file, being an interface, would probably also benefit from method-level comments.